app/views/projects/_dev_practices.erb

```diff
@@ -0,0 +1,46 @@
+<dl class='row detail-card dev-practices'>
+  <h2 class='col-xs-12'>
+    Development practices
+  </h3>
+
+  <div class='gradient-blur'>
+    <%
+      [
+        'Source repo 2FA enabled',
+        'Package manager 2FA enabled',
+        'Is security responsive',
+        'Dependencies are managed',
+        'Issue-free release available',
+        'Succession plan available',
+        'Package manager 2FA enable'
+      ].each do |practice|
+    %>
+      <dt class='col-xs-9'>
+        <%= practice %>
+      </dt>
+      <dd class='col-xs-3 blurry'>
+        <b>TEXT!</b>
+      </dd>
+    <%end%>
+  </div>
+  <div class='col-xs-8'>
+    <b>+ 12 more</b>
+  </div>
+  <div class='col-xs-12 has-margin-top'>
+    <div class='has-margin-top'>
+      <%= link_to image_tag('Tidelift-logo-on-light.svg', width: 160), 'https://tidelift.com/?utm_source=librariesio&utm_medium=referral&utm_campaign=dev-practices' %>
+    </div>
+    <p class='tidelift-text has-margin-top'>
+      The Tidelift Subscription provides access to a continuously curated stream of <b>human-researched and maintainer-verified data</b> on open source packages and their licenses, releases, vulnerabilities, and development practices.
+    </p>
+    <a
+      type='button'
+      href='https://tidelift.com/?utm_source=librariesio&utm_medium=referral&utm_campaign=dev-practices'
```

Comment:
Can this go to https://tidelift.com/solutions/open-source-data ?

Reply:
yep!